src/Parallel/Invoke.hpp

```diff
@@ -49,18 +49,9 @@ template <
 void receive_data(Proxy&& proxy, typename ReceiveTag::temporal_id temporal_id,
                   ReceiveDataType&& receive_data,
                   const bool enable_if_disabled = false) noexcept {
-  auto* obj = proxy.ckLocal();
-  // Only elide the Charm++ RTS if the object is local and if we won't blow the
-  // stack by having too many recursive function calls.
-  if (obj != nullptr and not detail::max_inline_entry_methods_reached()) {
-    obj->template receive_data<ReceiveTag>(
-        std::move(temporal_id), std::forward<ReceiveDataType>(receive_data),
-        enable_if_disabled);
-  } else {
-    proxy.template receive_data<ReceiveTag>(
-        std::move(temporal_id), std::forward<ReceiveDataType>(receive_data),
-        enable_if_disabled);
-  }
+  proxy.template receive_data<ReceiveTag, std::decay_t<ReceiveDataType>>(
```

Comment:
why do you need to specify the second template parameter now?

Reply:
This is a frustrating result of the charm definition file changes when `[inline]` is included. For some reason, in one of the places it appears, it adds template parameters for all of the arguments, making the original two template arguments not inferrable unless explicitly specified.

Reply:
ugh; oh well